zkevm-circuits/src/super_circuit.rs

```diff
@@ -400,6 +404,14 @@ impl<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize, const MOCK_RANDO
             Value::known(block.randomness),
         )?;
 
+        let rpi_bytes = self.pi_circuit.public_data.get_pi_bytes(
+            block.circuits_params.max_txs,
+            block.circuits_params.max_calldata,
+        );
+        config
+            .keccak_table
+            .dev_load(&mut layouter, vec![&rpi_bytes], &challenges)?;
+
```

Comment:
Thanks for the reminding! 
To move it to witness generation part, it require quite of refactor to avoid code duplication, because in `keccak_inputs` there is no `public_data` object (defined in `pi_circuit`), and right now raw pi bytes are populated by 
`public_data get_pi_bytes()`. Will think about how to do it properly

Reply:
Done in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1345/commits/b891356f8cf6d8f1f5d7946bfa2a9ae906a9ece0